repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tasks/ClassificationTask.java

```diff
@@ -93,6 +92,8 @@ public AtlasTask.Status perform() throws AtlasBaseException {
         RequestContext.get().setUser(userName, null);
 
         try {
+            setStatus(IN_PROGRESS);
```

Comment:
set task status to in progress in class AbstractTask itself
* This method does some extra work of removing task guid from entity vertex
* Adding to AbstractTask.run() will take effect for every run method which overrides this class e.g. MeaningsTask

Reply:
Understood why it is called here, resolving this comment

Reply:
addressed MeaningsTask